NwPluginAPI/Enums/ServerEventType.cs

```diff
@@ -398,13 +406,13 @@ public enum ServerEventType : int
         /// </remarks>
         RagdollSpawn = 47,
 
-        /// <summary>
-        /// Executed when player throws item.
-        /// </summary>
-        /// <remarks>
-        /// Parameters: <see cref="IPlayer"/> player, <see cref="Item"/> item.
-        /// </remarks>
-        PlayerThrowItem = 48,
+		/// <summary>
+		/// Executed when player throws item.
+		/// </summary>
+		/// <remarks>
+		/// Parameters: <see cref="IPlayer"/> player, <see cref="Item"/> item, <see cref="ThrowableItem.ProjectileSettings"/> projectileSettings.
```

Comment:
As far as I know there are no ProjectileSettings in this case. It's a regular throwing, not throwing a throwable item (eg. a grenade).